packages/astro/test/astro-component-bundling.test.js

```diff
@@ -45,7 +45,8 @@ describe('Component bundling', () => {
 			await fixture.build();
 		});
 
-		it('should treeshake FooComponent', async () => {
+		// Not sure how this can possibly work, we bundle the module as an entrypoint.
+		it.skip('should treeshake FooComponent', async () => {
```

Comment:
```suggestion
		it('should treeshake FooComponent', {skip: "Not sure how this can possibly work, we bundle the module as an entrypoint."}, async () => {
```

What do you think of this way of skipping the test?